tests/generator/Makefile

```diff
@@ -10,7 +10,7 @@ build-unit-tests:
 
 run-unit-tests: build-unit-tests
 	rm -f .failed-stamp
-	$(SYSTEM_MONO) --debug $(XIBUILD_EXE_PATH) -t -- $(TOP)/packages/NUnit.ConsoleRunner.3.9.0/tools/nunit3-console.exe $(abspath $(TOP)/tests/generator/bin/Debug/generator-tests.dll) "--result=$(abspath $(CURDIR)/TestResult.xml);format=nunit2" $(TEST_FIXTURE) --labels=All || touch $(CURDIR)/.failed-stamp
+	$(TOP)/tools/nunit3-console-3.11.1 $(abspath $(TOP)/tests/generator/bin/Debug/generator-tests.dll) "--result=$(abspath $(CURDIR)/TestResult.xml);format=nunit2" $(TEST_FIXTURE) --labels=All || touch $(CURDIR)/.failed-stamp
```

Comment:
Seems easy to miss one/some
Maybe make `$(TOP)/tools/nunit3-console-3.11.1` a variable that is reused everywhere its needed ?

Reply:
It depends on the NUnitLite version defined in the csproj for each test project, which complicates matters.

A thought I had (just after implementing this fix) was to make a generic `$(TOP)/tools/nunit3-console` script that takes the csproj as an argument, and parses it to determine what to do. In the end I decided to postpone this until the next time the problem shows up.